iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp

```diff
@@ -40,6 +40,70 @@ TEST_F(RoudiFindService_test, OfferSingleMethodServiceSingleInstance)
     ASSERT_THAT(*instanceContainer.begin(), Eq(IdString("instance1")));
 }
 
+TEST_F(RoudiFindService_test, OfferServiceWithDefaultDescription)
+{
+    senderRuntime->offerService(iox::capro::ServiceDescription());
+    this->InterOpWait();
+    iox::runtime::InstanceContainer instanceContainer;
+    receiverRuntime->findService(iox::capro::ServiceDescription(), instanceContainer);
+
+    ASSERT_THAT(instanceContainer.size(), Eq(1u));
+    senderRuntime->stopOfferService(iox::capro::ServiceDescription());
+}
+
+TEST_F(RoudiFindService_test, ReOfferSingleMethodServiceSingleInstance)
+{
+    senderRuntime->offerService({"service1", "instance1"});
```

Comment:
This is maybe more than one test? StopOffering and ReOfffering?

Reply:
i dint split test here as i m checking for scenario  where service might get stopped and start again

Reply:
@nihalchari I think @budrus meant a test for:
- that stopOffer works
- that offered and service after it was stopped works (which is your current test)

Reply:
okay, test for stopOffer works is present at https://github.com/eclipse/iceoryx/blob/master/iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp #148